ipatests/test_integration/test_otp.py

```diff
@@ -83,13 +85,72 @@ def kinit_otp(host, user, *, password, otp, success=True):
     )
 
 
+def ssh_2fa_with_cmd(host, username, password, otpvalue, command="exit 0"):
+    """ ssh to user and in same session pass the command to check tgt of user
+    :param host: host to ssh
+    :param str username: The name of user
+    :param str password: password, usually the first factor
+    :param str otpvalue: generated pin of user
+    :param str command: command to execute in same session,
+     by deafult set to "exit 0"
+    :return: object class of expect command run
+    """
+    host.run_command('dnf install -y expect')
```

Comment:
As the tests can also be run on debian or ubuntu, the code should be compatible with other package managers. You can use 
```
from ipatests.pytest_ipa.integration import tasks
tasks.install_packages(host, ['expect'])
```

Reply:
updated.